app/models/student.rb

```diff
@@ -0,0 +1,13 @@
+class User < ActiveRecord::Base
+  attr_accessible :title, :first_name, :middle_name, :last_name, :email, :birth_date, :gender end
+
+  # allow for null values where applicable to allow for single names
+  # allow for null values where applicable since you cannot legally force a user to identify their gender
+  validates :title, length: {maximum => 10 }
+  validates :last_name, :length: { maximum => 50 } , presence: { message: "Please enter your family name" }
+  validates :first_name, :middle_name, :length: { maximum => 50 }
+  validates :email, presence: { message: "Please enter your email address"  }, length: {maximum => 100}
```

Comment:
I think there are some syntax issues with the `maximum => 10` fields, it should be one of:

```ruby
{ maximum: 10 }
```

OR

```ruby
{ :maximum => 10 }
```

The first way is the "newer" (since like 2010) ruby object syntax (sometimes called json syntax), and the second way is referred to has the "hash rocket" syntax.

In JR plus we prefer the first approach (the "newer" ruby object syntax) and since your other parameters follow this lets use that way. There's no real difference aside from aesthetics, but I think it's important to keep consistency

Reply:
hmmm. not really sure where I picked up that notation. Once I put it in I copied it to the other fields.